spec/integration/puppetfile_spec.rb

```diff
@@ -8,17 +8,28 @@
   include BoltSpec::Integration
 
   let(:synced_modules) { [] }
-  let(:root) { Dir.mktmpdir }
-  let(:boltdir) { File.join(root, 'Boltdir') }
-  let(:module_source) { Dir.mktmpdir }
+  let(:boltdir) { File.join(@root, 'Boltdir') }
+
+  # Usually temp dirs would just be created under the system temp dir,
+  # but this doesn't work on Windows images on GitHub Actions. Creating a
+  # temp dir in the current working dir gets around this issue.
+  around(:each) do |example|
+    Dir.mktmpdir(nil, Dir.pwd) do |root|
```

Comment:
Can this just be `let(:root) { Dir.mktmpdir(nil, Dir.pwd) }` instead of being an around(:each)? mktmpdir doesn't require a block

Reply:
It can, I believe it just requires an `after(:each)` to clean up the tmpdir. Using `mktmpdir` with a block will clean up the tmpdirs automatically.